cmake/AddCA.cmake

```diff
@@ -730,6 +732,9 @@ endif()
       can be specified, each must be of the form: "VAR=<value>"
 #]=======================================================================]
 function(add_ca_check name)
+  if (NOT CA_ENABLE_TESTS)
+    return()
+  endif()
```

Comment:
This makes the function no longer match its documentation. I think the change makes sense if you update the documentation too though.